applications/app/controllers/GalleryController.scala

```diff
@@ -43,7 +43,7 @@ object GalleryController extends Controller with RendersItemResponse with Loggin
       .showFields("all")
     ).map { response =>
       val gallery = response.content.map(Content(_))
-      val model: Option[GalleryPage] = gallery collect { case g: Gallery => GalleryPage(g, RelatedContent(g, response), index, isTrail) }
+      val model: Option[GalleryPage] = gallery collect { case g: Gallery => GalleryPage(g, StoryPackages(g, response), index, isTrail) }
```

Comment:
In my previous change I had anticipated the rename of the `RelatedContent` object into `StoryPackages` by duplicating some code. See https://github.com/guardian/frontend/commit/0425a079e3af26b61c3fb1b36cd243312ca511fa#diff-9b3d36cd982208831cf027d918e19a9cL26

Now I am cleaning up the 2 different a/b versions, I can go forward with this renaming (which make more sense)